variables.tf

```diff
@@ -580,6 +586,12 @@ variable "enable_waf" {
   default     = false
 }
 
+variable "is_pv_encryption_in_transit_enabled" {
```

Comment:
As above, please use the imperative style. Also, next (before or after) the key id.